src/lib/tooltip/tooltip.ts

```diff
@@ -35,11 +35,9 @@ import {
   OverlayConnectionPosition,
   OriginConnectionPosition,
 } from '../core';
+import {Directionality, Platform, first} from '@angular/cdk';
 import {Observable} from 'rxjs/Observable';
 import {Subject} from 'rxjs/Subject';
-import {Directionality} from '../core/bidi/index';
-import {Platform} from '../core/platform/index';
-import {first} from '../core/rxjs/index';
 import {ScrollDispatcher} from '../core/overlay/scroll/scroll-dispatcher';
 import {coerceBooleanProperty} from '@angular/cdk';
```

Comment:
I think this can be merged to another import:

`import {coerceBooleanProperty, Directionality, Platform, first} from '@angular/cdk';`